ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/dropdown/v1/dropdown/clientlibs/editor/js/editDialog.js

```diff
@@ -124,5 +125,30 @@
         }
     );
 
+    /**
+     * Enables the auto-complete functionality for the dropdown component.
+     * @param {HTMLElement} dialog The dialog on which the operation is to be performed.
+     */
+    function handleAutoComplete (dialog) {
+        var autoComplete = dialog.find(DROPDOWN_AUTOCOMPLETE)[0];
```

Comment:
This code isn't necessary. Why do you want to add the datalist dynamically? It seems like you were testing something and forgot to remove it.